tools/regression_test.py

```diff
@@ -1070,24 +1118,28 @@ def main():
         report_txt_path = report_save_path.with_suffix('.txt')
 
         report_dict = {
-            'model_name': [],
-            'model_config': [],
-            'task_name': [],
-            'model_checkpoint_name': [],
-            'dataset': [],
-            'backend_name': [],
-            'deploy_config': [],
-            'static_or_dynamic': [],
-            'precision_type': [],
-            'conversion_result': [],
-            'fps': []
+            'Model': [],
+            'Model Config': [],
+            'Task': [],
+            'Checkpoint': [],
```

Comment:
for ncnn we may only add `xxx.param` path as ckpt.

Reply:
Done